PicApp_git/PicApp/app/Http/Controllers/Auth/LoginController.php

```diff
@@ -37,4 +38,8 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+    protected function loggedOut(Request $request)
```

Comment:
logoutで良さそうです！
```suggestion
    protected function logout(Request $request)
```

と思ったんですが、これって今使ってますか？

Reply:
こちら消し忘れになります（汗）
消しておきます。